src/quill.imageDrop.js

```diff
@@ -72,6 +72,15 @@ export class ImageDrop {
     if (images.length === 0) {
       return;
     }
+
+
+    // Text pasted from word will contain both text/html and image/png. 
+    // 
+    if (Array.from(evt.clipboardData.items).some(f => f.type === 'text/html')) {
```

Comment:
Can we filter out the items with `f.type === 'text/html'`? then we could still paste some images. Not a big deal if it doesn't work though

Reply:
I think that's done already here -- https://github.com/benwinding/quill-image-compress/blob/master/src/quill.imageDrop.js#L70

The issues with this are twofold: 
a) the "image" from Word shouldn't be pasted, and 
b) because this module handles the paste (but just for the image) the html is lost